internal/data/sql/queries/metrics.sql

```diff
@@ -149,13 +149,14 @@ SELECT validator_index, status FROM validator_status
 LIMIT $1 OFFSET $2;
 
 -- name: CreateProposerDuties :exec
-WITH data (public_key, validator_index, slot) AS (
+WITH data (public_key, validator_index, slot, epoch) AS (
```

Comment:
Why do we need to add epochs here? Isn't this redundant information (epoch = slot / epoch_length)?

Is there a specific example where it makes sense to have this in here?

Reply:
yes, i actually didnt add it until my last commit. The reason to add it is because it makes our queries simpler. Shutter explorer basically requires to query this table, if we didnt add epochs to it, we would need to pass 16 slots in the query to get all proposers for 1 epoch. But now since we have epochs available I can just pass the epoch number and fetch all 16 proposers active during those 16 slots i.e 1 epoch

Reply:
fixed